### PR TITLE
Extract and refactor Annotator Toolbar SCSS

### PR DESCRIPTION
**Pull request by lyzadanger**
Part of #2520

This PR continues toward completion of CSS organization in the client. It extracts out styling for the `Toolbar` component in the annotator and refactors the SCSS per current conventions.

This PR also moves annotator component SASS modules into a `components` subdirectory to match organization conventions for the `sidebar` side of the app.

User-visible changes: The two toolbar buttons (show highlights, create page note) have moved down about ~1px as standard layout multiples are used (via `ems`) are used here for margins.

### CODE REVIEW

File: src/annotator/components/toolbar.js [hunk 1]
```diff
@@ -121,21 +120,21 @@ export default function Toolbar({
...
-      )}
-      {!useMinimalControls && (
-        <ToolbarButton
-          label={
-            newAnnotationType === 'note' ? 'New page note' : 'New annotation'
-          }
-          icon={newAnnotationType === 'note' ? 'note' : 'annotate'}
-          onClick={createAnnotation}
-        />
+        <div className="annotator-toolbar-buttonbar">
```
Comment (lyzadanger) [inline src/annotator/components/toolbar.js:123]: This change is to add a parent/wrapper element that can handle the spacing of the buttons (margins).

**Review by LMS007 — APPROVED** [review]
Changes look good, just a few questions/comments below if you could address?
I can't visually review this PR because my mac is not working atm. I need to deal with this today, but if visually the only change is that 1px, then ship it!

File: src/annotator/components/toolbar.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lyzadanger) [reply src/annotator/components/toolbar.js:123]: In this case, I felt like this buttonbar was independent enough as a little visual module to warrant its own fully-qualified block classname.

File: src/annotator/components/toolbar.js [hunk 2]
```diff
@@ -104,15 +103,15 @@ export default function Toolbar({
     <div>
```
[the remark this answers is not in the document]
Comment (lyzadanger) [reply src/annotator/components/toolbar.js:103]: I don't know that we have a set standard!

**Pull request merged**